lib/rubocop/config_loader.rb

```diff
@@ -205,8 +205,8 @@ def dirs_to_search(target_dir)
       end
 
       def old_auto_config_file_warning
-        warn Rainbow('Attention: rubocop-todo.yml has been renamed to ' \
-                     "#{AUTO_GENERATED_FILE}").red
+        warn Rainbow('Attention: rubocop-todo.yml has been renamed to ' +
```

Comment:
Are you thinking the cop should not report an offense for this code? I can get on board with that. Essentially thinking of the code as `"Attention: rubocop-todo.yml has been renamed to #{AUTO_GENERATED_FILE}"`which doesn't violate the rule. That would also make autocorrect much simpler again.